tooling/autodispose-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "java"
```

Comment:
`java-library`, then use `api` and `implementation` configurations in dependencies

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/a9bf40217b0916c4bfa759f85ef8dd86c92e63fd